codespell_lib/data/dictionary.txt

```diff
@@ -4487,9 +4488,16 @@ bulletted->bulleted
 bult->built
 bult-in->built-in
 bultin->builtin
-bumb->bump
+bumb->bump, bomb,
```

Comment:
Perhaps suggest bum too, which is what I'd expect for the homeless person or the bottom.